Blockzilla/BrowserViewController.swift

```diff
@@ -281,6 +281,8 @@ class BrowserViewController: UIViewController {
         alertController.addAction(UIAlertAction(title: UIConstants.strings.copyImage, style: .default) { _ in copyAction() })
         alertController.addAction(UIAlertAction(title: UIConstants.strings.cancel, style: .cancel))
 
+        alertController.popoverPresentationController?.sourceView = self.view
```

Comment:
Just wondering - this code seems to apply to all form factors and not just iPad? I may be missing something?

Reply:
It doesn't change the behavior on my 7+. I'm reading up on it now just to be sure.